### ISSUE TITLE
FixIT+ transcripts reverting back to original status

### ISSUE THREAD

**Issue by mivillesvik** [WGBH-MLA/transcript-editor#144]
**Describe the bug**
Hi, 

One of the Kansas transcribers has noted an issue when she edits transcripts in the FixIT+. She is currently an admin, but has noticed that after she edits a transcript, upon going back and checking the transcript in question, some (not all) of the lines have changed back to a previous edit, and not what she had input. 

It should be noted that other Kansas transcribers haven't reported a similar issue; I'm not sure if it's something about her system, poor timing (maybe she was editing at the same time as another admin-Kansas transcriber), or something else! 
We're also kind of wondering if it's something she's doing, although I wouldn't be sure how...
In short, we're not 100% sure what the issue is, or if there even is an issue, but it would be great if someone can give this a look into!

**To Reproduce**

Not sure how to reproduce the error, since I haven't experienced it myself. I guess edit a couple lines as admin and see what happens.

**Expected behavior**

I would expect the transcript to retain the admin person's edits!

**Screenshots**

**Desktop (please complete the following information):**
 - Browser: Chrome

**Comment by RobesByPierre** [comment]
Hi there,

Sorry, I'm actually the Kansas transcriber with this problem! Any help you all might be able to provide would be fantastic. 

Here's some background info in case this is useful - I have been working on the Kansas files for a couple of months just on my own, but just got Admin status recently. Basically, there are a handful of "completed" transcripts that were done partially by me and partially by another volunteer. These few have errors throughout that need to be fixed for the transcript to be usable. I assumed with Admin status I could just hop on there, make the correction, and the file would be ready to go. Again, keep in mind that all the files I'm talking about in this case are ones that are fully "dark green" and technically completed.

Unfortunately, I'm running into a strange problem. I spent several hours working on corrections to a couple of files, which I will link. But when I went back to review them, only about half of my changes had saved. The others had, seemingly in a random pattern, reverted back to incorrect text. 

On this particular file (Artsnotes '88), the top half was originally completed by someone else and has quite a few errors. As you can see in the first few lines, there is a female interviewee whose name is supposed to be spelled Ossie E. Tranbarger. But it was originally spelled ?Aussie Trenbergar? (question marks included) and variations thereof. When I try to correct all instances of this misspelling, along with many other errors throughout the document, only some changes will stick. I've run into the same problem on the other correction files as well.

LINK:

https://fixitplus.americanarchive.org/transcripts/cpb-aacip_836e5cb7520

OK, I'm really grasping at straws here, so please ignore this part if it makes no sense. 

Here is my amateur guess as to what's happening. As you know, there is an algorithm in the software that "selects" the best transcript provided by volunteers by rating it. The algorithm judges lines that have more punctuation and capital letters and ellipses as "better." 

Now obviously this algorithm is not useful at all in the editing stage, as presumably the editor/admin (in this case me) knows exactly where he wants to remove or place punctuation and capitals, etc. Although I do have Admin access, the system seems to be somehow still rating my corrections and deciding whether or not to save them based on this algorithm. 

From what I understand from other software, sometimes this problem can be solved by increasing the "Hierarchy Number" assigned to a particular user. For example, if I was coded as a "5", perhaps a "4" designation would grant me permission to override this algorithm completely in my edits as needed. As for now, I'm unable to make any further corrections to these files, as they simply don't save. 

**Just as an FYI, I tried to make sure this wasn't something on my end - I tried switching browsers, flagging the line first and reloading the page before making my corrections, deleting the entire word first, deleting the entire line, etc. Nothing made any difference. Occasionally, a change would "stick" for say 20 minutes, and then disappear.** 

Since presumably everyone here has admin status, please feel free to go to the linked file above to see/experiment with what I'm struggling with. If it would be easier to talk offline, that's no problem either. I'm Marian on the Admin list, btw. 

Thank you so much for your help, please let me know what other information I can provide. Naturally if it's something I'm inadvertently doing myself, I'll be hideously embarrassed for wasting your time, but still grateful to be able to move forward on these files! :)

Thanks again!

Marian S.
